### PR TITLE
feat: add ongoing events filter

### PR DESCRIPTION
**Pull request by mrtraser**

### CODE REVIEW

**Comment by mrtraser** [comment]
@sanderland, is generator fix makes sense here?

File: cognite/client/_api/events.py [hunk 1]
```diff
@@ -238,10 +241,13 @@ def list(
             asset_subtree_ids = self._process_ids(asset_subtree_ids, asset_subtree_external_ids, wrap_ids=True)
         if data_set_ids or data_set_external_ids:
             data_set_ids = self._process_ids(data_set_ids, data_set_external_ids, wrap_ids=True)
+        if end_time and ("max" in end_time or "min" in end_time) and "isNull" in end_time:
+            raise TypeError("isNull cannot be used with min or max values")
```
Comment (sanderland) [inline cognite/client/_api/events.py:245]: i think  this is a ValueError no?

Comment (mrtraser) [reply cognite/client/_api/events.py:245]: think you right, changing

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,10 @@ Changes are grouped as follows
 - Separate read/write fields on data classes
 
 ## Unreleased
+### Added
+- Support events filtering by ongoing events (events without `end_time` defined)
+- Support events filtering by active timerange of event
+- Support files metadata filterign by `asset_external_ids`
```
Comment (polomani) [inline CHANGELOG.md:26]:
```suggestion
- Support files metadata filtering by `asset_external_ids`
```

Comment (mrtraser) [reply CHANGELOG.md:26]: will be fixed with https://github.com/cognitedata/cognite-sdk-python/pull/673

**Pull request merged**